specs-go/v1/config.go

```diff
@@ -45,6 +45,9 @@ type ImageConfig struct {
 
 	// WorkingDir sets the current working directory of the entrypoint process in the container.
 	WorkingDir string `json:"WorkingDir,omitempty"`
+
+	// Labels contains arbitrary metadata for the container.
+	Labels map[string]struct{} `json:"labels,omitempty"`
```

Comment:
The spec calls for [`map[string]string`][1].

And we'll need JSON Schema work too, although that doesn't have to happen in this PR.

[1]: https://github.com/opencontainers/image-spec/pull/371/files#diff-c9c91c29b41257aea3a3403cc606ad99R139